SinglePointTransform_lib/Transformation.cpp

```diff
@@ -40,12 +39,11 @@ float convertAngleToRad(int deg_angle)
     float ext_angle = deg_angle * 3.14159 / 180;
     return ext_angle;
 }
-
-QPointF transformPoint(QPointF rotCenter, QPointF point, float module,
-                       float angle)
+QPointF transformPoint(QPointF rotCenter, float module, float angle)
```

Comment:
From implementation I see name of this function is not precise.
Finally this function converts polar coordinates of point to XY coordinates of point and after that offset this.

It could looks like:
```cpp
QPoint i_point = fromPolar(module, angle) + rotCenter
```

Reply:
I have doubt if function calcModule works properly. What if angle is equal `x*180 + 90` ?

Reply:
Corrected.